airflow-core/src/airflow/ui/src/pages/DagsList/DagTags.tsx

```diff
@@ -31,7 +32,11 @@ type Props = {
 export const DagTags = ({ hideIcon = false, tags }: Props) => (
   <LimitedItemsList
     icon={hideIcon ? undefined : <FiTag data-testid="dag-tag" />}
-    items={tags.map(({ name }) => name)}
+    items={tags.map(({ name }) => (
+      <RouterLink key={name} to={`/dags?tags=${name}`}>
```

Comment:
If you have more than 3 tags, the rest will be displayed in a tooltip. We might want to see if passing `interactive` to the tooltip will be enough to make those ones linkable too

Reply:
Thanks for the catch. I have made `interactive` to be opt in and on hover it just display the tags with a separator. Then user can select the relevant tag. Also removed an extra comma which resulted in ", , +2 more" which seems to be not correct.

![image](https://github.com/user-attachments/assets/9aa1de64-b4f4-434c-8e44-25d72f019de1)

Reply:
nice!